sdk/search/Azure.Search.Documents/tests/Samples/Tests/RecursiveModel.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma warning disable SA1402 // File may only contain a single type
+#pragma warning disable SA1649 // File name should match first type name
+
+namespace Azure.Search.Documents.Tests.Samples
```

Comment:
Nit - these are just for tests, right?  Might be good to pull these out of the Samples namespace to make it clear.

Reply:
Given the other namespace change I made to pull samples out of "Tests", I made these all "...Samples.Tests" and moved them out of the folder to avoid confusion as well.